### PR TITLE
Fix invalid home locations giving blank errors

### PR DESCRIPTION
**Pull request by JRoy**

### CODE REVIEW

**Review by TheIntelloBox — APPROVED** [review]

**Comment by Octol1ttle** [comment]
@TheIntelloBox There's no need to approve pull requests unless you have write access (and unless you previously requested changes on the same PR), because it does not affect mergeability and triggers a (slightly annoying) notification

**Review by mdcfe — APPROVED** [review]

File: Essentials/src/main/java/com/earth2me/essentials/commands/Commandhome.java
Comment (rafaelsms) [inline Essentials/src/main/java/com/earth2me/essentials/commands/Commandhome.java:84]: @JRoy if the first home is from a deleted world, players can't see their homes on /homes because of this condition

**Review by rafaelsms — COMMENTED** [review]
(I believe this is a regression related to #4055)

File: Essentials/src/main/java/com/earth2me/essentials/commands/Commandhome.java
Comment (JRoy) [reply Essentials/src/main/java/com/earth2me/essentials/commands/Commandhome.java:84]: Thanks for spotting this, I'll get this fixed sooner than later.

**Comment by TNTUP** [comment]
Found out some of the worlds looks like "176a8ad6-aa96-4459-83e7-0be437ec3049" which causes the issue "player has not set a home" but I never created such world. How EssX determines the correct world? That UUID named world should bring to the "world"  which is somewhat confusing as hell :thokingintensifies:

**Comment by JRoy** [comment]
> Found out some of the worlds looks like "176a8ad6-aa96-4459-83e7-0be437ec3049" which causes the issue "player has not set a home" but I never created such world. How EssX determines the correct world? That UUID named world should bring to the "world"  which is somewhat confusing as hell :thokingintensifies:

All worlds are given a UUID and are independent of names (so names can change and the teleport doesn't break). You can see a worlds UUID in their level.dat

Additionally, any place where you see a UUID in data files, you can manually put world names there too and Essentials will resolve it to the world's UUID.

**Pull request merged**
